### PR TITLE
Scaffolding jdwp  for new JDWP debugger project

### PR DESCRIPTION
**Pull request by wekesa360**
This pull request addresses the issue #53, however I'm currently facing an issue with running the placeholder targets e.g `buck2 run //projects/jdwp:main`. An error is thrown as 
```
From `load` at implicit location

Caused by:
    0: From `load` at prelude/prelude.bzl:8:6-29
    1: From `load` at prelude/native.bzl:15:6-45
    2: From `load` at prelude/apple/apple_macro_layer.bzl:14:5-36
    3: From `load` at prelude/apple/apple_rules_impl_utility.bzl:9:6-46
    4: From `load` at prelude/apple/apple_bundle_types.bzl:8:6-18
    5: From `load` at prelude/apple/debug.bzl:9:5-35
    6: Error evaluating module: `prelude//artifact_tset.bzl`
    7: Traceback (most recent call last):
         * prelude/artifact_tset.bzl:29, in <module>
             _tset = field([_ArtifactTSet, None], None),
       error: Found `_ArtifactTSet` instead of a valid type annotation. Perhaps you meant `"_ArtifactTSet"`?
         --> prelude/artifact_tset.bzl:29:13
          |
       29 |     _tset = field([_ArtifactTSet, None], None),
          |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
          |
       
Build ID: 0a784514-076a-44d5-80c9-42d81616c2e8
Jobs completed: 3. Time elapsed: 0.2s.
BUILD FAILED

```

### CODE REVIEW

**Comment by michalgr** [comment]
Hi !

The issue you're seeing is caused by incompatible versions of buck2 and 'prelude'  module. It should go away if you rebase past https://github.com/facebookexperimental/ExtendedAndroidTools/commit/35f3021bfcdad0f95fdfe1528e6e11208be95fb6.

File: .buckconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+fbsource = none
+buck = none
+
+[parser]
+target_platform_detector_spec = target:root//...->prelude//platforms:default
+
+[project]
+ignore = .git
+
+[python]
```
Comment (michalgr) [inline .buckconfig:19]: Is this section added by you or was the entire file generated ? I would hope this is not necessary

File: BUCK [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# A list of available rules and their signatures can be found here: https://buck2.build/docs/api/rules/
```
Comment (michalgr) [inline BUCK:1]: this file can be deleted

File: projects/jdwp/BUCK [hunk 3]
```diff
@@ -0,0 +1,11 @@
+jdwp_binary(
```
Comment (michalgr) [inline projects/jdwp/BUCK:1]: Buck does not know what `jdwp_binary` is, but you can tell it to build parts of python project with the following rules:
- [`python_library`](https://github.com/facebook/buck2/blob/main/examples/with_prelude/python/library/BUCK) 
- [`python_binary`](https://github.com/facebook/buck2/blob/main/examples/with_prelude/python/hello_world/BUCK#L3C1-L3C14)
- `python_test` (see "Examples" [here](https://buck.build/rule/python_test.html))

File: toolchains/BUCK [hunk 4]
```diff
@@ -0,0 +1,6 @@
+load("@prelude//toolchains:genrule.bzl", "system_genrule_toolchain")
+
+system_genrule_toolchain(
```
Comment (michalgr) [inline toolchains/BUCK:3]: we will not need this, we will need to setup python toolchain [like this](https://github.com/facebook/buck2/blob/95dd7908111ed529637a321b0a365573e6b5cc30/prelude/toolchains/demo.bzl#L40-L48)

File: .buckconfig (hunk 1 above)
Comment (wekesa360) [reply .buckconfig:19]: This file was generated, I only added the last two lines, for the interpreter and the pyre paths.

Comment (wekesa360) [reply .buckconfig]: I have deleted the additional lines.

File: BUCK (hunk 2 above)
Comment (wekesa360) [reply BUCK]: I have deleted the file

File: projects/jdwp/BUCK (hunk 3 above)
Comment (wekesa360) [reply projects/jdwp/BUCK]: I have made some changes, and the targets run without errors or any blocker

**Comment by michalgr** [comment]
Looks good !

I noticed that pyre does not work yet, but we can handle that later.

Could you add the following copyright to BUCK and *.py files ?
```
# Copyright (c) Meta Platforms, Inc. and affiliates.
```

**Pull request merged**